docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -231,6 +231,51 @@ docker run \
 
 The default Intel Quote Provider config is available in [Intel SGX Github repository](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/qcnl/linux/sgx_default_qcnl.conf).
 
+## Migrate from EPID Attestation to DCAP Attestation
+
+EPID attestation will be discontinued in 2025 and will no longer be available on
+any processors. All nodes using EPID attestation should migrate to DCAP
+attestation.
+
+:::info
+
+Compute node operators, please hold for further instructions. We will coordinate
+the migration in phases to ensure that during the migration the network remains
+operational.
+
+:::
+
+:::info
+
+In case you are running AESM service on Docker please see [these instructions].
```

Comment:
The docker part affects only steps `3.`, `4.` and `5.`. If running AESM service in docker other steps are still required (e.g. checking for DCAP attestation support, gracefully shutting down the node, configuring the quote provider, testing with attestation tool...)
  
Not sure how to nicely incorporate this, but it should be something like this:
```
1)...
2)...
3) -  If running AESM in docker -> follow [these instructions] 
     -  Otherwise manually Configure AESM service to use [DCAP attestation]: Step 3, 4, 5 
4) Configure quote provider
5) Use attestation-tool...
...
```